app/javascript/components/upload/DifferentialExpressionStep.jsx

```diff
@@ -49,42 +49,68 @@ export function DifferentialFileUploadForm({
+                      <col className="col-genes" />
+                      <col className="col-logfoldchanges" />
+                      <col className="col-qval" />
+                      <col className="col-mean" />
+                      <col className="col-ellipsis" />
+                      <col className="col-logfoldchanges" />
+                      <col className="col-qval" />
+                      <col className="col-mean" />
+                      <col className="col-ellipsis" />
+                      <col className="col-logfoldchanges" />
+                      <col className="col-qval" />
+                      <col className="col-mean" />
+                      <col className="col-ellipsis" />
+                      <col className="col-logfoldchanges" />
+                      <col className="col-qval" />
+                      <col className="col-mean" />
+                      <col className="col-ellipsis" />
+                    </colgroup>
+                    <thead>
+                      <tr><td>genes</td><td><span className="blue">A</span>--<span className="red">rest</span>--logfoldchanges</td><td><span className="blue">A</span>--<span className="red">rest</span>--qval</td><td><span className="blue">A</span>--<span className="red">rest</span>--<span className="optional">mean</span></td><td>...</td><td><span className="green">B</span>--<span className="red">rest</span>--logfoldchanges</td><td><span className="green">B</span>--<span className="red">rest</span>--qval</td><td><span className="green">B</span>--rest--<span className="optional">mean</span></td><td>...</td><td>A--<span className="green">B</span>--logfoldchanges</td><td><span className="blue">A</span>--<span className="green">B</span>--qval</td><td><span className="blue">A</span>--<span className="green">B</span>--<span className="optional">mean</span></td><td>...</td><td><span className="blue">A</span>--<span className="yellow">C</span>--logfoldchanges</td><td><span className="blue">A</span>--<span className="yellow">C</span>--qval</td><td><span className="blue">A</span>--<span className="yellow">C</span>--<span className="optional">mean</span></td><td>...</td></tr>
+                    </thead>
+                    <tbody>
+                      <tr><td>It2ma</td><td>0.00049</td><td>0.00009</td><td>6.00312</td><td>...</td><td>-3.00246</td><td>0.00000</td><td>0.51128</td><td>...</td><td>-0.10246</td><td>0.40019</td><td>0.41357</td><td>...</td><td>0.00249</td><td>0.00103</td><td>0.42130</td><td>...</td></tr>
+                      <tr><td>Sergef</td><td>-0.00036</td><td>0.00239</td><td>4.20466</td><td>...</td><td>0.00036</td><td>0.074825</td><td>12.71389</td><td>...</td><td>0.00060</td><td>0.00005</td><td>1.82731</td><td>...</td><td>-0.00049</td><td>0.02648</td><td>1.06551</td><td>...</td></tr>
+                    </tbody>
+                  </table>
+                </div>
+                "Wide format" <i>does not</i> repeat values in the first column.
```

Comment:
Here `Wide format `has quotes around it, while on the other side `Long format `does not have quotes, it would probably be good to be consistent on one or the other

Reply:
Good catch!  Done in 45c114c.